person.rb

```diff
@@ -9,10 +9,11 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @name = name
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] Same here, Please add Getters and Setters(attr_accessor) for https://github.com/rentals instead Getters only. So the add rental method below can create a new rental.